scripts/subpath-workaround.mjs

```diff
@@ -41,7 +41,10 @@ async function run() {
 
   // Create directories for each subpath name using the pkgJsonPlaceholder
   subpathHelperFile.subpathNames.forEach(name => {
-    fs.mkdirSync(new URL(`../packages/${pkgName}/${name}`, import.meta.url));
+    const directoryURL = new URL(`../packages/${pkgName}/${name}`, import.meta.url);
+    if (!fs.existsSync(directoryURL)) {
+      fs.mkdirSync();
```

Comment:
You're missing the `directoryURL` inside mkdirSync.

For me the function exits early at line 32, did it not do that for you?